.github/workflows/push.yml

```diff
@@ -0,0 +1,25 @@
+name: GitHub Actions
+on: [ push ]
+env:
+  TEST_TENANT_ID: 9b85ee6f-4fb0-4a46-8cb7-4dcc6b262a89
+  TEST_CLIENT_ID: 04d6e451-e964-4cdf-ab2f-a6c77c3261fe
+  TEST_CLIENT_SECRET: ${{ secrets.TEST_CLIENT_SECRET }}
+  TEST_KEY_VAULT_URL: https://testing-object-encryptor.vault.azure.net/
+  TEST_AWS_REGION: us-east-1
+  TEST_AWS_ACCESS_KEY_ID: AKIAQLZBTO5VKBAGWLNC
```

Comment:
V jakem je to AWS accountu? premyslim kam podobne resourcy pro testy davat.

Reply:
025303414634

Reply:
jo `Dev - Platform Services`, ok to je zatim cajk. Pak si budeme mit pro ci veci samostatne accounty a ten dev bude jen na lokalni vyvoj a pokusy to ale pak dame vedet https://keboola.atlassian.net/wiki/spaces/KB/pages/2478604289/Migration+to+Control+Tower+SSO#Dal%C5%A1%C3%AD-kroky